plugins/controlled-documents-resources/src/utils.ts

```diff
@@ -714,7 +721,7 @@ export async function getLatestProjectId (spaceRef: Ref<DocumentSpace>): Promise
 
   if (space !== undefined) {
     if (space.$lookup?.type?.projects ?? false) {
-      const project = await getLatestProject(spaceRef)
+      const project = await getLatestProject(spaceRef, editableOnly)
```

Comment:
Should condition be inverted?

Reply:
no, that's just wrong name :)

Reply:
Ok, but why do we include readonly in document space presenter then? I thought we wanted to only show not readonly versions there? Or it's in another component?

Reply:
discussed